src/components/timeTable/TimeTable.js

```diff
@@ -42,6 +50,9 @@ const TimeTable = ({ stats, removeTime, times, showGraph, fixGraphYAxis }) => {
   const highlightedStat = stats.find(stat => stat.name === highlightedStatName) || {};
   const { includedIds = [], excludedIds = [] } = highlightedStat[highlightedStatType] || {};
 
+  const getSolveIndex = index => (showLatestSolveOnTop ? times.length - 1 - index : index);
+  const sortedTimes = showLatestSolveOnTop ? times.sort(createDescSorter('date')) : times;
```

Comment:
This would be clearer if it was more explicit and also did `times.sort(createAscSorter('date'))` for the `false` case. Though that is already what `times` is, so left it out.